src/components/offers/list/OffersView.js

```diff
@@ -58,9 +58,12 @@ export const OffersView = (props) => {
                                   offers={offers}/>}
             />}
         />
-    )
 }
 
 OffersView.propTypes = {
     getOffers: PropTypes.func.isRequired,
 }
+
+const OffersLoadingScreen = () => <div style={{display: "flex", justifyContent: "center", marginTop: "2em"}}>
```

Comment:
Jest to dokładnie to samo, co Loading Screen z wcześniejszego komponentu - wyrzuć do utilsów i korzystajmy z jednego - nie będzie potem, że tu większe, tu mniejsze. Jeśli większe / mniejsze było celowe, przekaż prop size.

Reply:
Spoko, a większe/mniejsze jest celowo, bo jeden się pokazuje na całej stronie, a drugi tylko w małej liście